packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -767,6 +767,16 @@ const CommentBoard = ( {
 			? actions.filter( ( item ) => item.isEligible( thread ) )
 			: [];
 
+	let deleteConfirmMessage = __(
+		'Are you sure you want to delete this note?'
```

Comment:
```suggestion
		'Are you sure you want to delete this reply?'
```
Should this say "reply" instead of "note" here? It technically is a reply to a note, even though it is a note under the hood. The submit button even says "Reply" in the interface.

<img width="285" height="281" alt="Image" src="https://github.com/user-attachments/assets/26473faa-aa0c-495d-9eaa-942c7e6f9294" />